README.md

```diff
@@ -1,4 +1,891 @@
 _Note that this project depends on internal Symphony infrastructure (repository.symphony.com), and therefore it can only be built by Symphony LLC employees/partners._
 
-# App-Integrations-Trello
-Trello Integration
+# Trello WebHook Integration
+The Trello WebHook Integration will allow you to receive notifications in Symphony whenever specific events, defined by you, occur in a Trello board of your choice.
+
+## [Build Instructions](#build-instructions-for-the-java-developer)
+
+## How it works
+As a Symphony user you can configure a WebHook with Trello directly from the Trello WebHook Application available on Symphony Market. There you can choose which events you want to be notified from Trello, which we'll cover in more details ahead.
+
+### Trello Authentication
+Unlike other integrations so far, when configuring Trello, we do not provide a URL to be pointed at on the referred application. As Trello only provides APIs to configure WebHooks and not a graphical interface, we created an interface where one can choose which events he wants to be notified by and we do the work calling the APIs to actually create a connection.
```

Comment:
Change from "choose which events he wants to be notified" to "choose which events one want to be notified" to keep the language the same across this document